docs/workflows/create_sync_publish.rst

```diff
@@ -146,9 +146,13 @@ Sync repository ``foo`` using remote ``bar``
 .. literalinclude:: ../_scripts/sync.sh
    :language: bash
 
-You can specify ``mirror=True`` for a mirror mode. It means Pulp won't update
-repository using previous repository version but create a new copy of remote
-repository as a new repository version.
+There are 3 sync modes to chose from, using the ``sync_policy`` option.
+
+- ``additive`` (the default) will retain the existing contents of the Pulp repository and add the contents of the remote repository being synced.
+- ``mirror_content_only`` will synchronize the Pulp repository to contain the same content as the one remote repository being synced - removing any existing content that isn't present in the remote repo.
+- ``mirror_complete`` will automatically create a publication that will be an _exact_ bit-for-bit copy of the remote repository being synced. This will keep metadata checksums intact, but is not possible for all repositories, as some use features which are incompatible with local clones.
```

Comment:
if you want to be fair in your statements you could say `This will keep  repo metadata checksums intact` this will narrow down the understanding towards primary, filelists, etc